### PR TITLE
feat(uie2e): Remove the dependence of tests on what is returned from /debug/provision

### PR DESCRIPTION
**Pull request by ebb-tide**
Tests will no longer depend on what is returned from `/debug/provision` endpoint.

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -16,15 +17,31 @@ export const signin = (): Cypress.Chainable<Cypress.Response> => {
...
+    .then(() =>
+      cy.request({
+        method: 'GET',
+        url: '/api/v2/orgs',
+      })
+    )
+    .then(response => {
+      cy.wrap(response.body.orgs[0]).as('org')
+    })
+    .then(() =>
```
Comment (ebb-tide) [inline cypress/support/commands.ts:36]: it might be a better idea to get `@bucket` only when it is needed by the test.

**Comment by drdelambre** [comment]
how are we keeping tokens in sync with secondary services (ie: telegraf) between oss and cloud across testing cycles without /debug/provision?

**Comment by drdelambre** [comment]
actually, what does this have to do with /debug/provision at all? i dont see anything wrong with the changes, just confused

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -16,15 +17,31 @@ export const signin = (): Cypress.Chainable<Cypress.Response> => {
...
   \*/
 
-  return cy.setupUser().then(body => {
-    return cy
-      .visit('/api/v2/signin')
-      .then(() => cy.get('#login').type(Cypress.env('username')))
-      .then(() => cy.get('#password').type(Cypress.env('password')))
-      .then(() => cy.get('#submit-login').click())
-      .then(() => cy.get('.theme-btn--success').click())
-      .then(() => cy.wrap(body))
```
Comment (ebb-tide) [inline cypress/support/commands.ts:26]: Previously--- all tests depend on what is returned from this 'body' returned from setupUser which is the response to a 'GET' request to `/debug/provision`

File: cypress/e2e/variables.test.ts [hunk 3]
```diff
@@ -1,11 +1,14 @@
+import {Organization} from '../../src/types'
+
 describe('Variables', () => {
   beforeEach(() => {
     cy.flush()
 
-    cy.signin().then(({body}) => {
-      cy.wrap(body.org).as('org')
-      cy.createQueryVariable(body.org.id)
-      cy.visit(`orgs/${body.org.id}/settings/variables`)
```
Comment (ebb-tide) [inline cypress/e2e/variables.test.ts:8]: All tests depend on org id returned by `debug/provision`.

**Comment by ebb-tide** [comment]
I removed the dependence of tests on what is returned from `debug/provision`. 
Previously, tests were getting orgID from response from `debug/provision`, but that was unnecessary because orgID can be got soo many other ways (that are actually in our API, even!)

**Comment by drdelambre** [comment]
gotcha.. i guess the confusion is in the mention of /debug/provision

here's the OSS specific version that defined the contract for reference:
https://github.com/influxdata/influxdb/blame/master/ui/cypress/support/commands.ts#L427-L436

/debug/provision is an endpoint that serves to abstract the implementation specific details of generating that intent so that we can run the same test suite against our other environment requirements. the aforementioned contract is forwarded the same as is for oss, and recreated using the same APIs as this PR for cloud e2e testing.

so this is effectively adding another set of api calls for oss, and moving the implementation on the cloud side into the cypress tests so that we can reduce the contract for what /debug/provision returns?

**Comment by ebb-tide** [comment]
You no longer need to have a contract for what `debug/provision` returns that different test environments need to fulfill. The test suite now depends entirely on what our actual API returns.

File: cypress/support/commands.ts (hunk 1 above)
Comment (ebb-tide) [reply cypress/support/commands.ts]: done

**Review by drdelambre — APPROVED** [review]
please change the title to something more fitting before merging, other than that the changes look alright

**Pull request merged**